deploy/example_clusters/cr-cluster-spark245.yaml

```diff
@@ -1,29 +1,38 @@
 #apiVersion: v1
 #kind: ConfigMap
 #metadata:
-#  name: jupyterhub
+#  name: ldapauthentication
 #  labels:
-#    kubedirector.hpe.com/cmType : jupyterhub_ldap_config
+#    kubedirector.hpe.com/cmType : ldap_config
 #data:
-#  jupyterhub_server_address: "16.143.20.82"
-#  jupyterhub_server_port: "389"
-#  jupyterhub_lookup_dn: "True"
-#  jupyterhub_user_search_base: "dc=mip,dc=storage,dc=hpecorp,dc=net"
-#  jupyterhub_user_attribute: "cn"
-#  jupyterhub_bind_user_dn: "cn=admin,dc=mip,dc=storage,dc=hpecorp,dc=net"
+## auth_type options are LDAP/Active Directory
+#  auth_type: "Active Directory"
+#  ssl_enabled: "False"
+## use ',' as a delimiter to provide multiple server_address
+#  server_address: "10.10.10.100,10.10.10.101"
+#  ssl_enabled: "False"
+#  server_port: "389"
+#  lookup_dn: "True"
+#  user_search_base: "ou=people,dc=wikimedia,dc=org"
+#  user_attribute: "cn"
+#  bind_user_dn: "cn=admin,ou=people,dc=wikimedia,dc=org"
 ##  #optional parameter. "use ":" as a delimiter to provide multiple templates"
-#  jupyterhub_bind_dn_template: "cn={username},ou=People,dc=mip,dc=storage,dc=hpecorp,dc=net"
-#  jupyterhub_valid_username_regex: "^[a-z][.@a-z0-9_-]*$"
+#  bind_dn_template: "cn={username},ou=People,ou=people,dc=wikimedia,dc=org"
+#  valid_username_regex: "^[A-Za-z][.@A-Za-z0-9_-]*$"
+##  "use ":" as a delimiter to provide multiple member_of"
+#  member_of:
+#     - CN=Eng,OU=Engineering,ou=people,dc=wikimedia,dc=org
```

Comment:
update to this format in the above yaml file

Reply:
corrected.